docs/kpod-rmi.1.md

```diff
@@ -0,0 +1,27 @@
+## kpod-rmi "1" "March 2017" "kpod"
+
+## NAME
+kpod rmi - Removes one or more images.
+
+## SYNOPSIS
+**kpod** **rmi** **imageID [...]**
+
+## DESCRIPTION
+Removes one or more locally stored images.
+
+## OPTIONS
+
+**--force, -f**
+
+Forcibly remove the image
```

Comment:
Executing this command will stop all containers that are using the image and remove them from the system?